### ISSUE TITLE
Android build is broken after updating android sdk, jdk and gradle version

### ISSUE THREAD

**Issue by vinayseasia** [fastlane/fastlane#20363]
compileSdkVersion          : 32,
 minSdkVersion                 : 26,
 targetSdkVersion             : 32,

Gradle version                   : 7.3.3
Android Gradle Plugin      : 7.2.1
open jdk                             : 11 

Log file :  [apps_a8d2ed33cc464fd2_4c480e85-2aa9-49bc-a980-38fe02bd7398-full.txt](https://github.com/fastlane/fastlane/files/8854528/apps_a8d2ed33cc464fd2_4c480e85-2aa9-49bc-a980-38fe02bd7398-full.txt)

I am able to build my project on local machine(mac) but its failing on bitrise.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 12.3.1                                                         |
| Ruby                        | 2.6.8                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.33.0                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 12.3.1 (21E258)                                          |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Library/Developer/CommandLineTools/                           |
| Xcode Version               |                                                                |
| Swift Version               | 5.6.1                                                          |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./Fastfile`</summary>

```ruby
update_fastlane

default_platform(:android)

platform :android do
  desc "Build daily"
  lane :daily do
    gradle(
         task: 'bundle',
         build_type: 'daily',
         properties: {
               "android.injected.signing.store.file" => ENV["KEYSTORE_PATH"],
               "android.injected.signing.store.password" => ENV["KEYSTORE_PASSWORD"],
               "android.injected.signing.key.alias" => ENV["KEYSTORE_ALIAS"],
               "android.injected.signing.key.password" => ENV["PRIVATE_KEY_PASSWORD"],
          }
       )
    bundletool(
         ks_path: ENV["KEYSTORE_PATH"],
         ks_password: ENV["KEYSTORE_PASSWORD"],
         ks_key_alias: ENV["KEYSTORE_ALIAS"],
         ks_key_alias_password: ENV["PRIVATE_KEY_PASSWORD"],
         bundletool_version: "1.2.0",
         aab_path: "./app/build/outputs/bundle/daily/app-daily.aab",
         apk_output_path: "./app/build/outputs/bundle/daily/app-daily.apk",
         verbose: true
       )
    firebase_app_distribution(
         app: "1:746097776372:android:----",
         apk_path: "./app/build/outputs/bundle/daily/app-daily.apk",
         service_credentials_file: ENV["KEY_PATH"],
         groups: "testers"
    )
  end
end


```
</details>

<details><summary>`./Appfile`</summary>

```ruby
json_key_file("") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
package_name("") # e.g. com.krausefx.app

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.206.2 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                    | Version | Update-Status |
| ----------------------------------------- | ------- | ------------- |
| fastlane-plugin-bundletool                | 1.0.6   | ✅ Up-To-Date  |
| fastlane-plugin-firebase_app_distribution | 0.3.4   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                       | Version      |
| ----------------------------------------- | ------------ |
| did_you_mean                              | 1.6.1        |
| atomos                                    | 0.1.3        |
| claide                                    | 1.1.0        |
| colored2                                  | 3.1.2        |
| nanaimo                                   | 0.3.0        |
| rexml                                     | 3.2.5        |
| xcodeproj                                 | 1.21.0       |
| rouge                                     | 2.0.7        |
| xcpretty                                  | 0.3.0        |
| terminal-notifier                         | 2.0.0        |
| unicode-display_width                     | 1.8.0        |
| terminal-table                            | 1.8.0        |
| plist                                     | 3.6.0        |
| public_suffix                             | 4.0.7        |
| addressable                               | 2.8.0        |
| multipart-post                            | 2.0.0        |
| word_wrap                                 | 1.0.0        |
| optparse                                  | 0.1.1        |
| tty-screen                                | 0.8.1        |
| tty-cursor                                | 0.7.1        |
| tty-spinner                               | 0.9.3        |
| artifactory                               | 3.0.15       |
| babosa                                    | 1.0.4        |
| colored                                   | 1.2          |
| highline                                  | 2.0.3        |
| commander                                 | 4.6.0        |
| excon                                     | 0.92.3       |
| faraday-em_http                           | 1.0.0        |
| faraday-em_synchrony                      | 1.0.0        |
| faraday-excon                             | 1.1.0        |
| faraday-httpclient                        | 1.0.1        |
| faraday-multipart                         | 1.0.4        |
| faraday-net_http                          | 1.0.1        |
| faraday-net_http_persistent               | 1.2.0        |
| faraday-patron                            | 1.0.0        |
| faraday-rack                              | 1.0.0        |
| faraday-retry                             | 1.0.3        |
| ruby2_keywords                            | 0.0.5        |
| faraday                                   | 1.10.0       |
| unf_ext                                   | 0.0.8.2      |
| unf                                       | 0.1.4        |
| domain_name                               | 0.5.20190701 |
| http-cookie                               | 1.0.5        |
| faraday-cookie_jar                        | 0.0.7        |
| faraday_middleware                        | 1.2.0        |
| fastimage                                 | 2.2.6        |
| gh_inspector                              | 1.1.3        |
| mini_magick                               | 4.11.0       |
| naturally                                 | 2.2.1        |
| rubyzip                                   | 2.3.2        |
| security                                  | 0.1.3        |
| xcpretty-travis-formatter                 | 1.0.1        |
| dotenv                                    | 2.7.6        |
| bundler                                   | 2.3.15       |
| simctl                                    | 1.6.8        |
| jwt                                       | 2.4.1        |
| uber                                      | 0.1.0        |
| declarative                               | 0.0.20       |
| trailblazer-option                        | 0.1.2        |
| representable                             | 3.2.0        |
| retriable                                 | 3.1.2        |
| mini_mime                                 | 1.1.2        |
| memoist                                   | 0.16.2       |
| multi_json                                | 1.15.0       |
| os                                        | 1.1.4        |
| signet                                    | 0.16.1       |
| googleauth                                | 1.1.3        |
| httpclient                                | 2.8.3        |
| google-apis-core                          | 0.5.0        |
| google-apis-playcustomapp_v1              | 0.7.0        |
| google-apis-androidpublisher_v3           | 0.21.0       |
| google-cloud-env                          | 1.6.0        |
| google-cloud-errors                       | 1.2.0        |
| google-cloud-core                         | 1.6.0        |
| google-apis-iamcredentials_v1             | 0.10.0       |
| google-apis-storage_v1                    | 0.14.0       |
| digest-crc                                | 0.6.4        |
| google-cloud-storage                      | 1.36.2       |
| emoji_regex                               | 3.2.3        |
| jmespath                                  | 1.6.1        |
| aws-partitions                            | 1.597.0      |
| aws-eventstream                           | 1.2.0        |
| aws-sigv4                                 | 1.5.0        |
| aws-sdk-core                              | 3.131.1      |
| aws-sdk-kms                               | 1.57.0       |
| aws-sdk-s3                                | 1.114.0      |
| CFPropertyList                            | 3.0.5        |
| json                                      | 2.6.2        |
| webrick                                   | 1.7.0        |
| rake                                      | 13.0.6       |
| forwardable                               | 1.3.2        |
| logger                                    | 1.3.0        |
| date                                      | 3.2.2        |
| stringio                                  | 0.0.2        |
| openssl                                   | 2.1.2        |
| ipaddr                                    | 1.2.4        |
| zlib                                      | 1.0.0        |
| mutex_m                                   | 0.1.0        |
| ostruct                                   | 0.1.0        |
| strscan                                   | 1.0.0        |
| io-console                                | 0.5.11       |
| fileutils                                 | 1.6.0        |
| libxml-ruby                               | 3.2.3        |
| psych                                     | 3.1.0        |
| fastlane-plugin-bundletool                | 1.0.6        |
| fastlane-plugin-firebase_app_distribution | 0.3.4        |
</details>

*generated on:* **2022-06-08**
</details>

**Comment by Hendershott** [comment]
Just go to build setting setting and change Build system from gradle to internal. it will resolve your error.

**Comment by vinayseasia** [comment]
> It seems like you have not included the output of `fastlane env` To make it easier for us help you resolve this issue, please update the issue to include the output of `fastlane env` 👍


+------------------------------------------------------------------------------+
| (5) fastlane@2                                                               |
+------------------------------------------------------------------------------+
| id: fastlane                                                                 |
| version: 2.7.4                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2022-06-07T14:55:36Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
[34;1mConfig:
[0m- WorkDir: /bitrise/src
- Lane: daily
- UpdateFastlane: true
- VerboseLog: false
- EnableCache: true
- GemHome: 

[34;1mExpand WorkDir[0m
[32;1mExpanded WorkDir: /bitrise/src[0m

[34;1mDetermine desired Fastlane version[0m
[34;1mGem lockfile defined Fastlane version: 2.171.0[0m
[34;1mGem lockfile defined bundler version: 2.2.5[0m

[34;1mInstall bundler[0m
[32;1m$ gem "install" "bundler" "--force" "--no-document" "--version" "2.2.5"[0m

Successfully installed bundler-2.2.5
1 gem installed

[34;1mInstall Fastlane with bundler[0m
[32;1m$ bundle "_2.2.5_" "install" "--jobs" "20" "--retry" "5"[0m

Don't run Bundler as root. Bundler can ask for sudo if it is needed, and
installing your bundle as root will break this application for all non-root
users on this machine.
Fetching gem metadata from https://rubygems.org/........
Using rake 13.0.1
Using atomos 0.1.3
Fetching aws-partitions 1.398.0
Using babosa 1.0.4
Using bundler 2.2.5
Using colored 1.2
Using colored2 3.1.2
Using declarative 0.0.20
Using dotenv 2.7.6
Fetching emoji_regex 3.2.1
Using multipart-post 2.0.0
Fetching fastimage 2.2.0
Using gh_inspector 1.1.3
Using memoist 0.16.2
Using multi_json 1.15.0
Using httpclient 2.8.3
Using uber 0.1.0
Using retriable 3.1.2
Fetching highline 1.7.10
Fetching declarative-option 0.1.0
Fetching excon 0.78.0
Fetching ruby2_keywords 0.0.2
Fetching public_suffix 4.0.6
Fetching jmespath 1.4.0
Fetching claide 1.0.3
Fetching unf_ext 0.0.7.7
Fetching jwt 2.2.2
Fetching mini_mime 1.0.2
Fetching CFPropertyList 3.0.2
Using mini_magick 4.11.0
Fetching aws-eventstream 1.1.0
Fetching digest-crc 0.6.1
Fetching os 1.1.1
Fetching google-cloud-errors 1.0.1
Fetching plist 3.5.0
Fetching json 2.3.1
Installing fastimage 2.2.0
Installing jmespath 1.4.0
Installing CFPropertyList 3.0.2
Installing emoji_regex 3.2.1
Installing google-cloud-errors 1.0.1
Installing aws-eventstream 1.1.0
Installing declarative-option 0.1.0
Installing ruby2_keywords 0.0.2
Installing os 1.1.1
Installing claide 1.0.3
Installing excon 0.78.0
Installing plist 3.5.0
Installing highline 1.7.10
Using security 0.1.3
Installing jwt 2.2.2
Fetching rubyzip 2.3.0
Installing json 2.3.1 with native extensions
Fetching naturally 2.2.0
Installing mini_mime 1.0.2
Fetching slack-notifier 2.3.2
Using terminal-notifier 2.0.0
Installing aws-partitions 1.398.0
Installing digest-crc 0.6.1 with native extensions
Fetching unicode-display_width 1.7.0
Using tty-screen 0.8.1
Using tty-cursor 0.7.1
Using word_wrap 1.0.0
Using nanaimo 0.3.0
Using rouge 2.0.7
Fetching fastlane-plugin-bundletool 1.0.3
Fetching fastlane-plugin-firebase_app_distribution 0.3.2
Installing public_suffix 4.0.6
Installing slack-notifier 2.3.2
Installing rubyzip 2.3.0
Installing naturally 2.2.0
Fetching aws-sigv4 1.2.2
Fetching representable 3.0.4
Installing unicode-display_width 1.7.0
Installing unf_ext 0.0.7.7 with native extensions
Using tty-spinner 0.9.3
Using xcpretty 0.3.0
Fetching faraday 1.1.0
Installing fastlane-plugin-bundletool 1.0.3
Fetching xcpretty-travis-formatter 1.0.0
Using simctl 1.6.8
Installing fastlane-plugin-firebase_app_distribution 0.3.2
Installing aws-sigv4 1.2.2
Installing representable 3.0.4
Using terminal-table 1.8.0
Fetching xcodeproj 1.19.0
Installing xcpretty-travis-formatter 1.0.0
Fetching aws-sdk-core 3.109.3
Installing faraday 1.1.0
Installing xcodeproj 1.19.0
Fetching commander-fastlane 4.4.6
Installing aws-sdk-core 3.109.3
Fetching addressable 2.7.0
Installing commander-fastlane 4.4.6
Installing addressable 2.7.0
Fetching google-cloud-env 1.4.0
Fetching faraday_middleware 1.0.0
Installing google-cloud-env 1.4.0
Fetching signet 0.14.0
Installing faraday_middleware 1.0.0
Fetching google-cloud-core 1.5.0
Fetching aws-sdk-kms 1.39.0
Installing signet 0.14.0
Installing google-cloud-core 1.5.0
Installing aws-sdk-kms 1.39.0
Fetching googleauth 0.14.0
Fetching aws-sdk-s3 1.85.0
Installing googleauth 0.14.0
Fetching google-api-client 0.38.0
Installing aws-sdk-s3 1.85.0
Installing google-api-client 0.38.0
Using unf 0.1.4
Using domain_name 0.5.20190701
Fetching http-cookie 1.0.3
Installing http-cookie 1.0.3
Using faraday-cookie_jar 0.0.7
Fetching google-cloud-storage 1.29.1
Installing google-cloud-storage 1.29.1
Fetching fastlane 2.171.0
Installing fastlane 2.171.0
Bundle complete! 3 Gemfile dependencies, 75 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.

[34;1mFastlane version[0m
[32;1m$ bundle "_2.2.5_" "exec" "fastlane" "--version"[0m
fastlane installation at path:
/usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.171.0/bin/fastlane
-----------------------------
/usr/local/lib/ruby/gems/2.7.0/gems/highline-1.7.10/lib/highline.rb:624: warning: Using the last argument as keyword parameters is deprecated
fastlane 2.171.0

#######################################################################
# fastlane 2.206.2 is available. You are on 2.171.0.
# You should use the latest version.
# Please update using `bundle update fastlane`.
#######################################################################

[32m2.206.2 Improvements[0m
* [Fastlane.swift] fix compile issue with argumentProcessor (#20318) via Josh Holtz (@joshdholtz)
* [Fastlane.Swift] Use ArgumentProcessor port to start fastlane socket_server (#20176) via Sven Tiigi (@SvenTiigi)
* [pem] fixed "\x82" from ASCII-8BIT to UTF-8 when saving .p12 files to disk (#20317) via Roger Oba (@rogerluan)
* [fastlane] fix a grammatical mistake when prompting which lane to run (#20307) via kamimi01 (@kamimi01)
* [fastlane][tests] add tests for CLIToolsDistributor (#20315) via Lukasz Grabowski (@lucgrabowski)

[32m2.206.1 Hot fix[0m
* [regression][fastlane] fixed fastlane command issue when tool name not provided (#20295) via Manish Rathi (@crazymanish)
* [aciton][import_from_git] fix/import from git spec fail with non master default branch (#20297) via Jerome Lacoste (@lacostej)

[32m2.206.0 Improvements[0m
* [match] add support for Developer ID certificates from G2 Sub-CA (#20145) via Frederik Seiffert (@triplef)
* [spaceship] add ability to invite a single TestFlight user (#20112) via Lucas (@LcTwisk)
* [deliver] add `verify_only` option to deliver (#20247) via Pol Piella (@pol-piella)
* [scan] fix scan failing to return results when there are test failures (#20237) via Mahmood Tahir (@tahirmt)
* [docs] remove named parameter from `create_capability` example. (#20197) via Trent Kocurek (@t2)
* [spaceship] fix creation App Store Connect API authorization token (#20206) via Yuya Oka (@nnsnodnb)
* [match] add option to skip google account confirmation (#20223) via Tim Sneed (@trsneed)
* [aciton][setup_ci] added timeout param into setup_ci (#20211) via javigines (@javigines)
* [action][danger] add GitHub Enterprise flags (#20216) via Alexander Weiß (@alexanderwe)
* [fastlane] allows aliased tool names (build_app, sync_code_signing, etc) to run from CLI (#20287) via Josh Holtz (@joshdholtz)
* [spaceship] update buildDeliveries request to use app id in path (#20268) via Lukasz Grabowski (@lucgrabowski)
* [spaceship] update `Spaceship::ConnectAPI::User model` to include `delete!` method (#20251) via Liam Nichols (@liamnichols)
* [infra] changelog will now show github usernames which makes contributors stand out more in releases (#20214) via Josh Holtz (@joshdholtz)

[32mTo see all new releases, open https://github.com/fastlane/fastlane/releases[0m

[32mPlease update using `bundle update fastlane`[0m

[34;1mRun Fastlane[0m
[32;1m$ bundle "_2.2.5_" "exec" "fastlane" "daily"[0m
+-------------------------------------------+---------+----------------------------------------------+
|                                            [32mUsed plugins[0m                                            |
+-------------------------------------------+---------+----------------------------------------------+
| Plugin                                    | Version | Action                                       |
+-------------------------------------------+---------+----------------------------------------------+
| fastlane-plugin-bundletool                | 1.0.3   | bundletool                                   |
| fastlane-plugin-firebase_app_distribution | 0.3.2   | firebase_app_distribution_add_testers        |
|                                           |         | firebase_app_distribution                    |
|                                           |         | firebase_app_distribution_login              |
|                                           |         | firebase_app_distribution_get_udids          |
|                                           |         | firebase_app_distribution_get_latest_release |
|                                           |         | firebase_app_distribution_remove_testers     |
+-------------------------------------------+---------+----------------------------------------------+

[14:56:05]: Sending anonymous analytics information
[14:56:05]: Learn more at https://docs.fastlane.tools/#metrics
[14:56:05]: No personal or sensitive data is sent.
[14:56:05]: You can disable this by adding `opt_out_usage` at the top of your Fastfile
[14:56:05]: [32m-----------------------------[0m
[14:56:05]: [32m--- Step: update_fastlane ---[0m
[14:56:05]: [32m-----------------------------[0m
[14:56:05]: Looking for updates for fastlane...
[14:56:07]: Updating fastlane from [33m2.171.0[0m to [33m2.206.2[0m... 🚀
[14:56:49]: [32mFinished updating fastlane[0m
[14:56:49]: Cleaning up old versions...
[14:56:49]: fastlane.tools successfully updated! I will now restart myself... 😴
+-------------------------------------------+---------+----------------------------------------------+
|                                            [32mUsed plugins[0m                                            |
+-------------------------------------------+---------+----------------------------------------------+
| Plugin                                    | Version | Action                                       |
+-------------------------------------------+---------+----------------------------------------------+
| fastlane-plugin-bundletool                | 1.0.3   | bundletool                                   |
| fastlane-plugin-firebase_app_distribution | 0.3.2   | firebase_app_distribution_add_testers        |
|                                           |         | firebase_app_distribution                    |
|                                           |         | firebase_app_distribution_login              |
|                                           |         | firebase_app_distribution_get_udids          |
|                                           |         | firebase_app_distribution_get_latest_release |
|                                           |         | firebase_app_distribution_remove_testers     |
+-------------------------------------------+---------+----------------------------------------------+

[14:56:51]: [32m-----------------------------[0m
[14:56:51]: [32m--- Step: update_fastlane ---[0m
[14:56:51]: [32m-----------------------------[0m
[14:56:51]: [32m------------------------------[0m
[14:56:51]: [32m--- Step: default_platform ---[0m
[14:56:51]: [32m------------------------------[0m
[14:56:51]: [32mDriving the lane 'android daily' 🚀[0m
[14:56:51]: [32m-------------------------[0m
[14:56:51]: [32m--- Step: bundledaily ---[0m
[14:56:51]: [32m-------------------------[0m
[14:56:51]: [36m$ /bitrise/src/gradlew bundledaily -p . -Pandroid.injected.signing.store.file=/bitrise/src/engineering.keystore -Pandroid.injected.signing.store.password=[REDACTED] -Pandroid.injected.signing.key.alias=[REDACTED] -Pandroid.injected.signing.key.password=[REDACTED][0m
[14:56:51]: ▸ [35mDownloading https://services.gradle.org/distributions/gradle-7.3.3-bin.zip[0m
[14:56:53]: ▸ [35m............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................[0m
[14:56:53]: ▸ [35mUnzipping /root/.gradle/wrapper/dists/gradle-7.3.3-bin/6a41zxkdtcxs8rphpq6y0069z/gradle-7.3.3-bin.zip to /root/.gradle/wrapper/dists/gradle-7.3.3-bin/6a41zxkdtcxs8rphpq6y0069z[0m
[14:56:54]: ▸ [35mSet executable permissions for: /root/.gradle/wrapper/dists/gradle-7.3.3-bin/6a41zxkdtcxs8rphpq6y0069z/gradle-7.3.3/bin/gradle[0m
[14:56:55]: ▸ [35mWelcome to Gradle 7.3.3![0m
[14:56:55]: ▸ [35mHere are the highlights of this release:[0m
[14:56:55]: ▸ [35m- Easily declare new test suites in Java projects[0m
[14:56:55]: ▸ [35m- Support for Java 17[0m
[14:56:55]: ▸ [35m- Support for Scala 3[0m
[14:56:55]: ▸ [35mFor more details see https://docs.gradle.org/7.3.3/release-notes.html[0m
[14:56:55]: ▸ [35mStarting a Gradle Daemon (subsequent builds will be faster)[0m
[14:57:52]: ▸ [35m> Configure project :app[0m
[14:57:52]: ▸ [35mWarning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.[0m
[14:57:53]: ▸ [35mVINAY branchName AN-696-update-android-sdk-version[0m
[14:57:53]: ▸ [35m[[ --- BRANCH NAME --- ]][0m
[14:57:53]: ▸ [35mAN-696-update-android-sdk-version[0m
[14:57:53]: ▸ [35m[[ --- COMMIT MESSAGE --- ]][0m
[14:57:53]: ▸ [35mAN-696#version upgrades[android, jdk, gradle][0m
[14:57:53]: ▸ [35mAN-696#version upgrades[android, jdk, gradle, kotlin][0m
[14:57:53]: ▸ [35mAN-689 firebase crashlytics issue fix[0m
[14:57:53]: ▸ [35mMerged develop into AN-689-firebase-crashlytics-issues---4-6[0m
[14:57:53]: ▸ [35mAN-691#firebase crash fix[0m
[14:57:58]: ▸ [35m> Configure project :sdkApp[0m
[14:57:58]: ▸ [35mWarning: The 'kotlin-android-extensions' Gradle plugin is deprecated. Please use this migration guide (https://goo.gle/kotlin-android-extensions-deprecation) to start working with View Binding (https://developer.android.com/topic/libraries/view-binding) and the 'kotlin-parcelize' plugin.[0m
[14:58:17]: ▸ [35mWARNING: [Processor] Library '/bitrise/src/jitsi/com/facebook/react/react-native-webrtc/1.84.1-jitsi-6432623/react-native-webrtc-1.84.1-jitsi-6432623.aar' contains references to both AndroidX and old support library. This seems like the library is partially migrated. Jetifier will try to rewrite the library anyway.[0m
[14:58:17]: ▸ [35mExample of androidX reference: 'androidx/core/view/ViewCompat'[0m
[14:58:17]: ▸ [35mExample of support library reference: 'android/support/annotation/Nullable'[0m
[14:58:20]: ▸ [35m> Task :app:preBuild UP-TO-DATE[0m
[14:58:20]: ▸ [35m> Task :app:preDailyBuild UP-TO-DATE[0m
[14:58:20]: ▸ [35m> Task :sdkApp:preBuild UP-TO-DATE[0m
[14:58:20]: ▸ [35m> Task :sdkApp:preDebugBuild UP-TO-DATE[0m
[14:58:20]: ▸ [35m> Task :sdkApp:packageDebugRenderscript NO-SOURCE[0m
[14:58:20]: ▸ [35m> Task :app:compileDailyRenderscript NO-SOURCE[0m
[14:58:20]: ▸ [35m> Task :app:generateDailyResValues[0m
[14:58:20]: ▸ [35m> Task :app:generateDailyResources[0m
[14:58:20]: ▸ [35m> Task :app:injectCrashlyticsMappingFileIdDaily[0m
[14:58:20]: ▸ [35m> Task :app:processDailyGoogleServices[0m
[14:58:20]: ▸ [35m> Task :sdkApp:compileDebugRenderscript NO-SOURCE[0m
[14:58:20]: ▸ [35m> Task :sdkApp:generateDebugResValues[0m
[14:58:20]: ▸ [35m> Task :sdkApp:generateDebugResources[0m
[14:58:20]: ▸ [35m> Task :sdkApp:packageDebugResources[0m
[14:58:26]: ▸ [35m> Task :app:mergeDailyResources[0m
[14:58:26]: ▸ [35m/bitrise/src/app/src/main/res/values/strings.xml:74:4: Multiple substitutions specified in non-positional format of string resource string/button_views_clicks_count. Did you mean to add the formatted="false" attribute?[0m
[14:58:27]: ▸ [35m> Task :app:createDailyCompatibleScreenManifests[0m
[14:58:27]: ▸ [35m> Task :app:extractDeepLinksDaily[0m
[14:58:27]: ▸ [35m> Task :sdkApp:extractDeepLinksDebug[0m
[14:58:27]: ▸ [35m> Task :sdkApp:processDebugManifest[0m
[14:58:27]: ▸ [35m/bitrise/src/sdkApp/src/main/AndroidManifest.xml:36:5-125:19 Warning:[0m
[14:58:27]: ▸ [35mapplication@android:allowBackup was tagged at AndroidManifest.xml:36 to replace other declarations but no other declaration present[0m
[14:58:28]: ▸ [35m> Task :app:processDailyMainManifest[0m
[14:58:28]: ▸ [35m/bitrise/src/app/src/main/AndroidManifest.xml:471:5-77 Warning:[0m
[14:58:28]: ▸ [35mElement uses-permission#android.permission.FOREGROUND_SERVICE at AndroidManifest.xml:471:5-77 duplicated with element declared at AndroidManifest.xml:465:5-77[0m
[14:58:28]: ▸ [35m/bitrise/src/app/src/main/AndroidManifest.xml:6:5-445:19 Warning:[0m
[14:58:28]: ▸ [35mapplication@tools:name was tagged at AndroidManifest.xml:6 to replace other declarations but no other declaration present[0m
[14:58:29]: ▸ [35m> Task :app:processDailyManifest[0m
[14:58:29]: ▸ [35m> Task :app:processApplicationManifestDailyForBundle[0m
[14:58:30]: ▸ [35m> Task :sdkApp:compileDebugLibraryResources[0m
[14:58:30]: ▸ [35m> Task :app:mergeDailyShaders[0m
[14:58:30]: ▸ [35m> Task :app:compileDailyShaders NO-SOURCE[0m
[14:58:30]: ▸ [35m> Task :app:generateDailyAssets UP-TO-DATE[0m
[14:58:30]: ▸ [35m> Task :sdkApp:mergeDebugShaders[0m
[14:58:30]: ▸ [35m> Task :sdkApp:compileDebugShaders NO-SOURCE[0m
[14:58:30]: ▸ [35m> Task :sdkApp:generateDebugAssets UP-TO-DATE[0m
[14:58:30]: ▸ [35m> Task :sdkApp:packageDebugAssets[0m
[14:58:30]: ▸ [35m> Task :app:mapDailySourceSetPaths[0m
[14:58:32]: ▸ [35m> Task :app:mergeDailyAssets[0m
[14:58:33]: ▸ [35m> Task :app:buildKotlinToolingMetadata[0m
[14:58:33]: ▸ [35m> Task :sdkApp:compileDebugAidl NO-SOURCE[0m
[14:58:33]: ▸ [35m> Task :app:compileDailyAidl NO-SOURCE[0m
[14:58:33]: ▸ [35m> Task :app:generateDailyBuildConfig[0m
[14:58:33]: ▸ [35m> Task :sdkApp:writeDebugAarMetadata[0m
[14:58:33]: ▸ [35m> Task :app:checkDailyAarMetadata[0m
[14:58:33]: ▸ [35m> Task :app:processDailyManifestForPackage[0m
[14:58:34]: ▸ [35m> Task :sdkApp:parseDebugLocalResources[0m
[14:58:34]: ▸ [35m> Task :app:bundleDailyResources[0m
[14:58:35]: ▸ [35m> Task :sdkApp:generateDebugBuildConfig[0m
[14:58:35]: ▸ [35m> Task :sdkApp:javaPreCompileDebug[0m
[14:58:35]: ▸ [35m> Task :app:processDailyJavaRes NO-SOURCE[0m
[14:58:35]: ▸ [35m> Task :sdkApp:processDebugJavaRes NO-SOURCE[0m
[14:58:35]: ▸ [35m> Task :sdkApp:generateDebugRFile[0m
[14:58:44]: ▸ [35m> Task :app:checkDailyDuplicateClasses[0m
[14:58:44]: ▸ [35m[Fatal Error] :-1:-1: Invalid byte 1 of 1-byte UTF-8 sequence.[0m
[14:58:44]: ▸ [35mWARNING: [XmlResourcesTransformer] Received malformed sequence exception when trying to detect the encoding for 'search_url_text.xml'. Defaulting to UTF-8.[0m
[14:58:44]: ▸ [35mWARNING: [XmlResourcesTransformer] javax.xml.stream.XMLStreamException: ParseError at [row,col]:[1,1][0m
[14:58:44]: ▸ [35mMessage: Invalid byte 1 of 1-byte UTF-8 sequence.[0m
[14:58:44]: ▸ [35mat java.xml/com.sun.org.apache.xerces.internal.impl.XMLStreamReaderImpl.setInputSource(XMLStreamReaderImpl.java:242)[0m
[14:58:44]: ▸ [35mat java.xml/com.sun.org.apache.xerces.internal.impl.XMLStreamReaderImpl.<init>(XMLStreamReaderImpl.java:210)[0m
[14:58:44]: ▸ [35mat java.xml/com.sun.xml.internal.stream.XMLInputFactoryImpl.getXMLStreamReaderImpl(XMLInputFactoryImpl.java:262)[0m
[14:58:44]: ▸ [35mat java.xml/com.sun.xml.internal.stream.XMLInputFactoryImpl.createXMLStreamReader(XMLInputFactoryImpl.java:129)[0m
[14:58:44]: ▸ [35mat com.android.tools.build.jetifier.processor.transform.resource.XmlResourcesTransformer.getCharset(XmlResourcesTransformer.kt:106)[0m
[14:58:44]: ▸ [35mat com.android.tools.build.jetifier.processor.transform.resource.XmlResourcesTransformer.runTransform(XmlResourcesTransformer.kt:87)[0m
[14:58:44]: ▸ [35mat com.android.tools.build.jetifier.processor.Processor.visit(Processor.kt:549)[0m

**Comment by vinayseasia** [comment]
> Just go to build setting setting and change Build system from gradle to internal. it will resolve your error.

I can build on my local machine, but I'm having trouble building on Bitrise. I'm not sure where I should make changes.
